Chapter02/Activity2.01/app/build.gradle

```diff
@@ -1,15 +1,13 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
-    // Not added by default - add line below
-    id 'kotlin-android-extensions'
 }
 
 android {
     compileSdkVersion 30
 
     defaultConfig {
-        applicationId "com.example.myapplication"
+        applicationId "com.example.loginactivity"
```

Comment:
@drulabs - There are a series of my changes to update the app/build.gradle file which I noticed was wrong in all chapters